internal/app/splinter/model/interface.go

```diff
@@ -0,0 +1,58 @@
+package model
+
+type UserInterface interface {
```

Comment:
Could you exclude backend and model from this PR, please? It is not related to API. And too much to review in one PR